src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -81,6 +81,12 @@ export const AGGREGATIONS = {
     isSortable: true,
     multiPlotType: 'line',
   },
+  failure_count: {
+    parameters: [],
+    outputType: 'number',
+    isSortable: true,
+    multiPlotType: 'line',
```

Comment:
🤔 IMO this should be area like `count`

Reply:
Tbh, I'm not sure where this `multiPlotType` is used but I updated it to `'area'`.

Reply:
🥳  its used by the topEvents chart